src/components/mnemonic/mnemonic-create/mnemonic-create.tsx

```diff
@@ -45,8 +45,12 @@ export const MnemonicCreate: FC<MnemonicProps> = ({ value, isError, onChangeText
       />
       <View style={styles.buttonsContainer}>
         <ButtonSmallSecondary title="GEN NEW" onPress={handleGenerateNewButtonPress} />
-        <Divider size={formatSize(8)} />
-        <ButtonSmallSecondary title="COPY" onPress={() => copyStringToClipboard(value)} />
+        {value ? (
```

Comment:
`{isDefined(value) && <>...</>}`

Reply:
I'll check it a bit later... As far as I remember, the initial value is an empty string, so `isDefined` doesn't work here properly

Reply:
![Screenshot_2021-06-17_14-41-37](https://user-images.githubusercontent.com/10885399/122389950-25149e00-cf7a-11eb-9afe-83e8756ab60a.png)
Sorry, a single `isDefined` check won't work here properly
Then do we really need here something like `isDefined(value) && value.length > 0`?

Reply:
so, use `isString` istead of `isDefined` (it checks for empty string value)